README.md

```diff
@@ -1,2 +1,123 @@
 # Enact Starter Theme
 A starting theme for Enact developers to customize at will.
+
+## What's In It
+
+This theme is a minimal set of components, behaviors, and styles that can be used as a starting point to create a new theme as well as to start a new Enact project.
+
+### Components
+
+Most of the components in the starter theme are built from base components provided by [`@enact/ui`](https://github.com/enactjs/enact/tree/master/packages/ui).  The base components have minimal styling, just enough to ensure that the behavior and structure works, the rest is intentionally kept unassigned for the theme to define.
```

Comment:
Should we direct users to the source vs npm or enactjs.com?

Reply:
Personally, I appreciate that this refers to the source.

Reply:
OK